<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>638</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Street Planner</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout" columnstretch="0,1">
    <item row="6" column="0">
     <widget class="QPushButton" name="pushConfirm">
      <property name="text">
       <string>Confirm</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="labelFindRouteBetween">
      <property name="text">
       <string>Find Shortest Route Between</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QPushButton" name="pushAdd_Street">
      <property name="text">
       <string>Add Street</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" rowspan="19">
     <widget class="QGraphicsView" name="graphicsView"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="labelCity2">
      <property name="text">
       <string>City 2</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="labelCity1">
      <property name="text">
       <string>City 1</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QComboBox" name="comboCity1"/>
    </item>
    <item row="16" column="0">
     <widget class="QPushButton" name="pushTest_Draw_Map">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="toolTip">
       <string>Draw a map with 3 cities</string>
      </property>
      <property name="text">
       <string>Draw Map</string>
      </property>
     </widget>
    </item>
    <item row="18" column="0">
     <widget class="QPushButton" name="pushTest_Dijkstra">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Test Dijkstra</string>
      </property>
     </widget>
    </item>
    <item row="19" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="17" column="0">
     <widget class="QPushButton" name="pushTest_Draw_Street">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="toolTip">
       <string>Draw 3 streets connected between 3 cities</string>
      </property>
      <property name="text">
       <string>Draw Street</string>
      </property>
     </widget>
    </item>
    <item row="15" column="0">
     <widget class="QPushButton" name="pushTest_Draw_City">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="toolTip">
       <string>Draw two cities on the map</string>
      </property>
      <property name="text">
       <string>Draw City</string>
      </property>
     </widget>
    </item>
    <item row="14" column="0">
     <widget class="QPushButton" name="pushFill_Map">
      <property name="text">
       <string>Fill Map</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QComboBox" name="comboCity2"/>
    </item>
    <item row="11" column="0">
     <widget class="QCheckBox" name="checkTest_Mode">
      <property name="text">
       <string>Test Mode</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QPushButton" name="pushAdd_City">
      <property name="text">
       <string>Add City</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>638</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad_Map"/>
    <addaction name="actionSave_Map"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionAdd_City"/>
    <addaction name="actionAdd_street"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuTest">
    <property name="title">
     <string>Test</string>
    </property>
    <addaction name="actionTest_Draw_City"/>
    <addaction name="actionTest_Draw_Map"/>
    <addaction name="actionTest_Draw_Street"/>
    <addaction name="actionTest_Dijkstra"/>
   </widget>
   <widget class="QMenu" name="menuScene">
    <property name="title">
     <string>Scene</string>
    </property>
    <addaction name="actionClear_Scene"/>
   </widget>
   <widget class="QMenu" name="menuInfo">
    <property name="title">
     <string>Info</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuScene"/>
   <addaction name="menuTest"/>
   <addaction name="menuInfo"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAdd_City">
   <property name="text">
    <string>Add city</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionAdd_street">
   <property name="text">
    <string>Add street</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+N</string>
   </property>
  </action>
  <action name="actionFill_the_map_with_NRW">
   <property name="text">
    <string>Fill the map with NRW</string>
   </property>
  </action>
  <action name="actionClear_the_map">
   <property name="text">
    <string>Clear the map</string>
   </property>
  </action>
  <action name="actionLoad_Map">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Load Map</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave_Map">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Save Map</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Alt+E</string>
   </property>
  </action>
  <action name="actionClear_Scene">
   <property name="text">
    <string>Clear Scene</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionTest_Draw_City">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Test Draw City</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+1</string>
   </property>
  </action>
  <action name="actionTest_Draw_Map">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Test Draw Map</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+2</string>
   </property>
  </action>
  <action name="actionTest_Draw_Street">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Test Draw Street</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+3</string>
   </property>
  </action>
  <action name="actionTest_Dijkstra">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Test Dijkstra</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+4</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
